apps/cookbook/src/app/showcase/badge-showcase/badge-showcase.component.ts

```diff
@@ -9,13 +12,69 @@ import { ApiDescriptionProperty } from '~/app/shared/api-description/api-descrip
 export class BadgeShowcaseComponent {
   exampleHtml: string = require('!raw-loader!../../examples/badge-example/badge-example.component.html')
     .default;
+  _cardMode = 'flat';
+  _cardHasPadding = true;
 
-  properties: ApiDescriptionProperty[] = [
+  _properties: ApiDescriptionProperty[] = [
     {
       name: 'themeColor',
       description: 'Sets which color the badge should use from the theme palette.',
       defaultValue: 'white',
       type: ['white', 'success', 'warning', 'danger'],
     },
+    {
+      name: 'size',
+      description: 'Sets the size of the badge',
+      defaultValue: 'md',
+      type: ['sm', 'md'],
+    },
+  ];
+
+  _cssCustomPropertiesColumns: ApiDescriptionPropertyColumns = {
+    name: 'Name',
+    description: 'Description',
+    default: 'default',
+  };
+
+  _cssCustomProperties: ApiDescriptionProperty[] = [
+    {
+      name: '--kirby-badge-position',
+      description: 'Sets the position property of the badge.',
+      defaultValue: 'relative',
+    },
+    {
+      name: '--kirby-badge-top',
+      description: 'Sets the vertical position of the badge.',
+      defaultValue: 'auto',
+    },
+    {
+      name: '--kirby-badge-right',
+      description: 'Sets the horizontal position of the badge.',
+      defaultValue: 'auto',
+    },
+    {
+      name: '--kirby-badge-left',
+      description: 'Sets the horizontal position of the badge.',
+      defaultValue: 'auto',
+    },
+    {
+      name: '--kirby-badge-elevation',
+      description: 'Sets the box-shadow property of the badge.',
+      defaultValue: 'none',
+    },
+    {
+      name: '--kirby-badge-color',
+      description:
+        'Sets the text color of the badge. It is recommended to use the themeColor input property instead to not break the color scheme.',
+    },
+    {
+      name: '--kirby-badge-background-color',
+      description:
+        'Sets the background color of the badge. It is recommended to use the themeColor input property instead to not break the color scheme.',
+    },
+    {
+      name: '--kirby-badge-z-index',
```

Comment:
```suggestion
      name: '--kirby-badge-zindex',
```

I think the custom property is without the hyphen.

Reply:
Correct! However the actual property is called `z-index`. So I will go ahead and change it the other way around. 

It does not seem to be used in DRB and is only used a single place in the designsystem (avatar).

Reply:
Again, good call. Had not realised it is not used outside Kirbz.